packages/ui5-application-writer/templates/core/webapp/manifest.json

```diff
@@ -36,7 +36,7 @@
       "flexEnabled": true,
       "dependencies": {
         "minUI5Version": "<%- ui5.minUI5Version %>",
-        "libs": {<% ui5.ui5Libs.forEach((ui5Lib, idx, libs) => { %>
+        "libs": {<% ui5.ui5Libs.concat(ui5.customUi5Libs ?? []).forEach((ui5Lib, idx, libs) => { %>
```

Comment:
Is it ok to put this kind of logic in the templates? @IainSAP 
Should the `concat(ui5.customUi5Libs ?? [])` go in the relevant `.ts` file?

Reply:
@cianmSAP in general I am in agreement with you, it should not be in the template but in the ts file, however, in that case, I suggested it as a quick & easy solution that we don't need to introduce a new property on the config.
Nevertheless, I am of course not against changing that